forch/ports_testing_states_manager.py

```diff
@@ -0,0 +1,111 @@
+"""Module that manages the testing states of the access ports"""
```

Comment:
I think we need something other than "testing" as an adjective, it typically means something that's not this (unit tests, etc...).    I think we can just simplify everything to port_state rather than the full port_testing_state -- or need to find another adjective.  Also, I'd use the singular not plural, so ports->port and states->states. It's tricky but you "manage the state of a port"

Reply:
Changed the name of the class to PortStateManager, and the methods inside the class as well.